pyproject.toml

```diff
@@ -37,7 +37,8 @@ dependencies = [
     "Flask-Migrate==4.0.7",
     "Flask-SQLAlchemy>=3.0.3",
     "sqlalchemy-utils>=0.38.3",
-    "beautifulsoup4==4.12.3"
+    "beautifulsoup4==4.12.3",
+    "boto3==1.34.144"
```

Comment:
FSD utils is a library package,so it shouldn't pin specific versions of dependencies - it should support a range of versions to ensure it can be installed in a wide range of concrete applications which may require different versions of boto3.

So I'd suggest this should probably be boto3>=1.34. you could optionally also add <2 but there's pros and cons to that approach.

I'm aware we do specifically pin some package versions here, but I'd say that approach is more wrong than right.